src/Common/Testimonial/Testimonial.scss

```diff
@@ -0,0 +1,64 @@
+@import "../../variables";
+
+.testimonial {
+  background-color: $color3;
+
+  * {
+    color: $white;
+  }
+
+  .message-container {
+    position: relative;
+
+    .quote-start,
+    .quote-end {
+      font-size: 8rem;
+      font-family: Arial;
+      position: absolute;
+      transform-origin: 50% 50%;
+    }
+
+    .quote-start {
+      transform: rotate(180deg);
```

Comment:
To me, the start quote looks like an end quote and the end quote looks like a start quote.... Maybe we should shift this transform to `.quote-end`? What do you think? You must have had a reason for making them like this...